Documentation/network/egress-gateway.rst

```diff
@@ -67,13 +67,17 @@ the gateway node.
 Incompatibility with other features
 -----------------------------------
 
-Egress gateway is currently partially incompatible with L7 policies.
-Specifically, when an egress gateway policy and an L7 policy both select the same
-endpoint, traffic from that endpoint will not go through egress gateway, even if
-the policy allows it. Full support will be added in an upcoming release once
-:gh-issue:`19642` is resolved.
+* Egress gateway is currently partially incompatible with L7 policies.
+  Specifically, when an egress gateway policy and an L7 policy both select the same
+  endpoint, traffic from that endpoint will not go through egress gateway, even if
+  the policy allows it. Full support will be added in an upcoming release once
+  :gh-issue:`19642` is resolved.
 
-Egress gateway is not supported for IPv6 traffic.
+* Cilium must be configured to use Kubernetes for identity allocation
+  (``identityAllocationMode`` set to ``crd``) as egress gateway isn't
+  compatible with the ``kvstore`` mode. This is the default for new installations.
```

Comment:
Edits for clarity in format "Because `x`, then `y`", use active voice, address the user as "you"

This sentence isn't clear to me; please verify that my revision is correct. 
```suggestion
* Because the default in new Kubernetes installations for identity allocation (``identityAllocationMode`` set to ``crd``) isn't compatible with the ``kvstore`` mode, you must configure Cilium to use Kubernetes for identity allocation as egress gateway .
```

Reply:
@zacharysarah Thanks a lot for your review and suggestions!

Your second suggestion wasn't entirely correct anymore. Hence, I took your idea of "Because x, then y" and rephrased it once again. What do you think about this version?